packages/web/src/coh/types.ts

```diff
@@ -0,0 +1,19 @@
+interface CommanderData {
+    serverID: string;
+    commanderName: string;
+    description: string;
+    races: Array<string>;
+    abilities: Array<Record<string, any>>;
+}
+
+interface IntelBulletinData {
+    serverID: string;
+    bulletinName: string;
+    descriptionShort: string;
+    descriptionLong: string;
+    races: Array<string>;
+}
+
+type RaceName = "wermacht" | "usf" | "soviet" | "wgerman" | "british";
+
+export type { CommanderData, IntelBulletinData, RaceName };
```

Comment:
Tento soubor je dulezity, tady jsem presne definoval jak maji ty objekty vypada. 
Mohl jsem presne vyspecifikovat jeste ty abilities ale chtel jsem to nechat jako ANY protoze si rikal ze by si chtel pridat po tom jeste informace k tem abilitam tak jsem to nechtel svazat.